content/en/blog/2022/announcing-community-manager.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Announcing the OpenTelemetry Community Manager
+linkTitle: Announcing OpenTelemetry Community Manager
+date: 2022-09-24
+author: Austin Parker
```

Comment:
Point to your github profile (or twitter handle?) here

Reply:
Done